New deprecation warnings from recently added contrib modules

## Problem/Motivation
We decided to merge #281 (which added the Paragraphs module to Quickstart) and #324 (which added the Entity Embed module) without addressing the code deprecation warnings that our core/contrib deprecation check generates for the Paragraphs, Entity Reference Revisions, Feeds, and Entity Embed modules:

```
$ /src/az-quickstart-scaffolding/vendor/bin/phpstan analyse --configuration web/profiles/custom/az_quickstart/phpstan.neon web/modules web/themes

 ------ --------------------------------------------------------------------------------------------------- 
  Line   modules/contrib/entity_embed/src/EntityEmbedDisplay/EntityEmbedDisplayManager.php                  
 ------ --------------------------------------------------------------------------------------------------- 
  33     config:entity_view_mode_list cache tag might be unclear and does not contain the cache key in it.  
 ------ --------------------------------------------------------------------------------------------------- 

 ------ --------------------------------------------------------------------------------------------------------------- 
  Line   modules/contrib/entity_reference_revisions/src/Plugin/diff/Field/EntityReferenceRevisionsFieldDiffBuilder.php  
 ------ --------------------------------------------------------------------------------------------------------------- 
         Class Drupal\diff\FieldDiffBuilderBase not found.                                                              
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols                                             
  20     Class Drupal\diff\FieldDiffBuilderBase not found.                                                              
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols                                             
 ------ --------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------ 
  Line   modules/contrib/entity_reference_revisions/tests/src/Kernel/Plugin/migrate/destination/EntityReferenceRevisionsDestinationTest.php  
 ------ ------------------------------------------------------------------------------------------------------------------------------------ 
  66     Call to deprecated method readAttribute() of class PHPUnit\Framework\Assert:                                                        
         https://github.com/sebastianbergmann/phpunit/issues/3338                                                                            
  67     Call to deprecated method readAttribute() of class PHPUnit\Framework\Assert:                                                        
         https://github.com/sebastianbergmann/phpunit/issues/3338                                                                            
  99     Call to deprecated method readAttribute() of class PHPUnit\Framework\Assert:                                                        
         https://github.com/sebastianbergmann/phpunit/issues/3338                                                                            
  308    Call to deprecated method readAttribute() of class PHPUnit\Framework\Assert:                                                        
         https://github.com/sebastianbergmann/phpunit/issues/3338                                                                            
 ------ ------------------------------------------------------------------------------------------------------------------------------------ 

 ------ --------------------------------------------------------------------------------------------------------------------------------- 
  Line   modules/contrib/paragraphs/modules/paragraphs_library/tests/src/FunctionalJavascript/ParagraphsLibraryItemEntityBrowserTest.php  
 ------ --------------------------------------------------------------------------------------------------------------------------------- 
         Class Drupal\Tests\entity_browser\FunctionalJavascript\EntityBrowserWebDriverTestBase not found.                                 
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols                                                               
  14     Class Drupal\Tests\entity_browser\FunctionalJavascript\EntityBrowserWebDriverTestBase not found.                                 
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols                                                               
 ------ --------------------------------------------------------------------------------------------------------------------------------- 

 ------ -------------------------------------------------------------------- 
  Line   modules/contrib/paragraphs/src/Feeds/Target/Paragraphs.php          
 ------ -------------------------------------------------------------------- 
         Class Drupal\feeds\Feeds\Target\Text not found.                     
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols  
  21     Class Drupal\feeds\Feeds\Target\Text not found.                     
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols  
 ------ -------------------------------------------------------------------- 

                                                                                                                        
 [ERROR] Found 11 errors                                                                                                
```
## Proposed resolution
We need to figure out what to do about these warnings and determine if we can resolve them.

--- a/phpstan.neon
+++ b/phpstan.neon
@@ -7,13 +7,19 @@ parameters:
 		- '#Plugin definitions cannot be altered.#'
 		- '#Missing cache backend declaration for performance.#'
 		- '#Plugin manager has cache backend specified but does not declare cache tags.#'
-		# Ignore Symfony 4.3 Symfony\Component\EventDispatcher\Event deprecation errors (not fixable in Drupal 8.x).
+		# Ignore Symfony 4.3 deprecation errors (not fixable in Drupal 8.x).
 		- '#.*extends deprecated class Symfony\\Component\\EventDispatcher\\Event.*#'
 		- '#.*Fetching class constant class of deprecated class Symfony\\Component\\EventDispatcher\\Event.*#'
-		- '#.*has typehint with deprecated class Symfony\\[a-zA-Z0-9\\_]+\\Event.*#'
-		# Ignore certain PHPUnit 8 deprecation errors not fixable in Drupal 8.x.
+		- '#.*has typehint with deprecated (interface Symfony\\Component\\HttpFoundation\\File\\MimeType\\MimeTypeGuesserInterface|class Symfony\\[a-zA-Z0-9\\_]+\\Event).*#'
+		- '#Instantiation of deprecated class Symfony\\Component\\EventDispatcher\\Event.*#'
+		# Ignore certain PHPUnit 8 deprecation errors (not fixable in Drupal 8.x).
 		- '#Call to deprecated method setMethods\(\) of class PHPUnit\\Framework\\MockObject\\MockBuilder.*#'
 		- '#Call to deprecated method expectExceptionMessageRegExp\(\) of class PHPUnit\\Framework\\TestCase.*#'
+		- '#Call to deprecated method readAttribute\(\) of class PHPUnit\\Framework\\Assert.*#'
+		# Ignore missing classes from optional module dependencies.
+		- '#Class Drupal\\(Tests\\)?(diff|entity_browser|feeds).*not found#'
+		# Ignore entitiy_embed cache tag warning (see https://www.drupal.org/project/entity_embed/issues/3087572#comment-13307163).
+		- '#config:entity_view_mode_list cache tag might be unclear and does not contain the cache key in it.#'
 includes:
 	- %currentWorkingDirectory%/vendor/mglaman/phpstan-drupal/extension.neon
 	- %currentWorkingDirectory%/vendor/phpstan/phpstan-deprecation-rules/rules.neon

--- a/phpstan.neon
+++ b/phpstan.neon
@@ -7,12 +7,12 @@ parameters:
 		- '#Plugin definitions cannot be altered.#'
 		- '#Missing cache backend declaration for performance.#'
 		- '#Plugin manager has cache backend specified but does not declare cache tags.#'
-		# Ignore Symfony 4.3 deprecation errors (not fixable in Drupal 8.x).
+		# Ignore certain Symfony 4.3 deprecation errors not fixable in Drupal 8.x.
 		- '#.*extends deprecated class Symfony\\Component\\EventDispatcher\\Event.*#'
 		- '#.*Fetching class constant class of deprecated class Symfony\\Component\\EventDispatcher\\Event.*#'
 		- '#.*has typehint with deprecated (interface Symfony\\Component\\HttpFoundation\\File\\MimeType\\MimeTypeGuesserInterface|class Symfony\\[a-zA-Z0-9\\_]+\\Event).*#'
 		- '#Instantiation of deprecated class Symfony\\Component\\EventDispatcher\\Event.*#'
-		# Ignore certain PHPUnit 8 deprecation errors (not fixable in Drupal 8.x).
+		# Ignore certain PHPUnit 8 deprecation errors not fixable in Drupal 8.x.
 		- '#Call to deprecated method setMethods\(\) of class PHPUnit\\Framework\\MockObject\\MockBuilder.*#'
 		- '#Call to deprecated method expectExceptionMessageRegExp\(\) of class PHPUnit\\Framework\\TestCase.*#'
 		- '#Call to deprecated method readAttribute\(\) of class PHPUnit\\Framework\\Assert.*#'

